### DISCUSSION TITLE
rustdesk-1.2.4-x86_64.deb generates continuasly new IDs

### DISCUSSION THREAD

**Discussion by IT-Betreuung** [rustdesk/rustdesk#7520] [General]
The „rustdesk-1.2.4-x86_64.deb“
generates continuasly new IDs
(since when I don´t know).

I have noticed it on Debian Bookworm  6.1.0-18-amd64 - Debian 6.1.76-1
and Ubuntu Server 22.04.4 LTS - Linux 5.15.0-101-generic x86_64.

When I downgrade to „rustdesk-1.2.3-x86_64.deb“,
everything is O.K..

**Comment by rustdesk** [comment]
@IT-Betreuung please go to https://github.com/rustdesk/rustdesk/issues/7521
